sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClient.java

```diff
@@ -551,13 +558,16 @@ public Secret restoreSecret(byte[] backup) {
      * call {@link SecretClient#getSecret(SecretBase)} . This will return the {@link Secret} secrets with values included of the specified versions.</p>
      * {@codesnippet com.azure.security.keyvault.secretclient.listSecretVersions#string-Context}
      *
+     * <p><strong>Samples to iterate SecretBase by each page</strong></p>
```

Comment:
The sample description needs more description as to what its doing like the one above it, +  its doing it per page.